features/support/steps.js

```diff
@@ -91,10 +91,28 @@ When('the user clicks the save button', function () {
   });
 });
 
-Then(/the report (?:will be|is) displayed/, { timeout: 60 * 1000 }, function () {
-  // Wait for a second for the loading ind to disappear
-  return this.app.client.waitUntilWindowLoaded().getText('#output')
-    .should.eventually.not.equal('');
+When('the user clicks the settings button', function () {
+  return this.app.client.$('#appSettingsButton').then((element) => element.click());
+});
+
+When('the user selects {string} from the Default Gherkin Dialect drop-down menu', function (dialect) {
+  return this.app.client.$('#dialectSelection').then((element) => element.selectByVisibleText(dialect));
+});
+
+Then(/the report (?:will be|is) (displayed|hidden)/, { timeout: 60 * 1000 }, async function (reportStatus) {
```

Comment:
```suggestion
Then(/the report (?:will be|is) (displayed|hidden)/, { timeout: 60000 }, async function (reportStatus) {
```

Use a straight literal, no need for computation at runtime. If you need to do X Seconds, use a const.